tools/clang/test/CodeGenHLSL/quick-test/lib_select_res.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %dxc -T lib_6_1 %s | FileCheck %s

```

Comment:
Would be good to have a test that shows error message if the resource cannot be resolved at link time.